node/src/reactor/validator/tests.rs

```diff
@@ -181,3 +201,69 @@ async fn run_validator_network() {
     net.settle_on(&mut rng, is_in_era(2), Duration::from_secs(60))
         .await;
 }
+
+#[tokio::test]
+async fn run_equivocator_network() {
+    testing::init_logging();
+
+    let mut rng = crate::new_rng();
+
+    let alice_sk = SecretKey::random(&mut rng);
+    let size: usize = 2;
+    let mut keys: Vec<SecretKey> = (1..size).map(|_| SecretKey::random(&mut rng)).collect();
+    let mut stakes: BTreeMap<PublicKey, u64> = keys
+        .iter()
+        .map(|secret_key| (PublicKey::from(secret_key), 100))
+        .collect();
+    stakes.insert(PublicKey::from(&alice_sk), 1);
+    keys.push(alice_sk.clone());
+    keys.push(alice_sk);
+
+    let mut chain = TestChain::new_with_keys(&mut rng, keys, stakes);
+
+    let mut net = chain
+        .create_initialized_network(&mut rng)
+        .await
+        .expect("network initialization failed");
+
+    let is_in_era = |era_num| {
+        move |nodes: &Nodes| {
+            // check ee's era_validators against consensus' validators
+            let first_node = nodes.values().next().expect("need at least one node");
+
+            // Get a list of eras from the first node.
+            let expected_eras = era_ids(&first_node);
+
+            // Return if not in expected era yet.
+            if expected_eras.len() <= era_num {
+                return false;
+            }
+
+            // Ensure eras are all the same for all other nodes.
+            nodes
+                .values()
+                .map(era_ids)
+                .all(|eras| eras == expected_eras)
+        }
+    };
+
+    println!("Waiting for Era 1 to end");
```

Comment:
should remove println's

Reply:
These are in the test. They are pretty helpful for me to understand how far the test has gone, since it takes ~50 seconds on my machine.  However if you think they are too chatty I can remove them.

Reply:
You could e.g. turn them into `info!` logs. Then they're only printed if the test is run with `RUST_LOG=info`.

Reply:
I've switched to `info!` in https://github.com/CasperLabs/casper-node/pull/615/commits/b81295e4b428737874d45b21ed1f9e7ca9a8b08a